.github/runlint.py

```diff
@@ -29,24 +29,28 @@ def main(pr):
     packages = set()
     for line in diff.split("\n"):
         if line.startswith("+++ b/recipes/") or line.startswith("--- a/recipes/"):
-            packages.add(line.split("/")[2])
-    for package in packages:
-        version = packaging.version.Version("0.0.0")
-        folder = ""
+            parts = line.split("/")
+            if len(parts) >= 5:
```

Comment:
why 5 ?

Reply:
`+++ b/recipes/foo/config.yml` would be 4, which we want to ignore,
`+++ b/recipes/foo/all/anyfile.ext` would be 5, which clearly identifies a recipe folder, so we can handle it